src/components/DoctorConnectSheet/UserCard.tsx

```diff
@@ -111,54 +113,97 @@ export default function UserCard({ user: facilityUser }: UserCardProps) {
                   <TooltipContent>{userFullName}</TooltipContent>
                 </Tooltip>
               </TooltipProvider>
-
-              <div className="text-xs font-medium">
+              <div className="text-sm font-light flex gap-2 mt-1 items-center">
+                <UserRound className="h-3 w-3" />
                 {facilityUser.role.name}
               </div>
+              <div className="flex items-center text-sm font-light text-muted-foreground mt-1 gap-2">
+                <CalendarDays className="h-3 w-3" />
```

Comment:
```suggestion
                <CalendarDays className="size-3" />
```

do it  rest of places there are many left 😅